Yoast/yoast-components
# 10310 implement content optimalisation container

## Summary

This PR can be summarized in the following changelog entry:

* Created a container for the content optimisation tab.  

## Relevant technical choices:

* The chosen implementation assumes the individual components will function independently, and take care of their own data. This container only determines whether or not these components should be shown. 
* The components that need to be shown in this tab are not finished yet: the two current components are placeholders. 

## Test instructions

This PR can be tested by following these steps:

* Checkout this branch.
* Go to localhost:3333, there is a new tab: Content optimization container. Make sure the `SnippetEditorExample` & `ContentAnalysisWrapper` are show correctly here. 

Fixes #[10310](https://github.com/Yoast/wordpress-seo/issues/10310)
